Updating CallRecords API documentation - New fields added (#20859)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/callrecords-session.md

|endDateTime|DateTimeOffset|UTC time when the last user left the session. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`|
|failureInfo|[microsoft.graph.callRecords.failureInfo](callrecords-failureinfo.md)|Failure information associated with the session if the session failed.|
|id|string|Unique identifier for the session. Read-only.|

> |isTest|Boolean|True if session correspond to a test.|

Reviewer: Suggested description: Specifies if the session is a test.
And if there's a default value,  The default value is _______.

> |isTest|Boolean|True if session correspond to a test otherwise false.|

Reviewer: ```suggestion
|isTest|Boolean|Specifies if the session is a test.|
```
